concepts/documentation-strategy/webiste-engine-poc.md

```diff
@@ -0,0 +1,97 @@
+
+Additionally, [Kyma modularization](../modularization/) initiated changes to hosting all Kyma documentation in a central repository. The modular approach assumes that each module repository includes module documentation. As a result the content needs to be pulled from different repositories which is not an easy [`webiste`] feature to implement.
+
+The decision on the tooling choice may be influenced by the fact that the Kyma project is also a foundation of SAP BTP, Kyma runtime which is a part of SAP Business Technology Platform (BTP). The SAP BTP, Kyma runtime documentation is hosted on the SAP [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud).
+
+## Potential solutions
+
+To address the challenge, we should find and choose tooling to publish open-source Kyma documentation. Considering different options, take into account the following requirements:
+
+- user experience
+- concise and user-friendly domain
+- intuitive linking
+- search feature
+- right-side navigation displaying the document's headers (to avoid a really, really long left-side navigation node)
+- analytics feature
+
+### SAP Help Portal
+
+The SAP BTP, Kyma runtime documentation is hosted on the SAP [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud). One of the potential solutions is hosting open-source documentation using the same engine. Documentation on the Help Portal is created using the DITA CCMS.
+
+Potential setup:
+
+- open-source documentation generally available on the `Cloud Production` level ??
+- ??
+- internal documentation available on the `Internal draft` level ??
+
+#### Advantages:
+
+- The Help Portal provides the possibility of having different versions of documentation depending on the user type by ading flags and using different containers.
+- Having all Kyma documentation in Help Portal would enusre a unified user experince and only one entry point for Kyma users.
+- Becoming an author requires license and training which gives more controle over the product documentation.
+- Analytics and serach options enabled.
+- Unified UI and URL.
+- Intuitive navigation.
+- Blog feature enabled.
+- What's New feature enabled.
+
+#### Disadvantages:
+
+- DITA CCMS is slow and requires relatively big effort to add and publish content.
+- DITA CCMS is not a developer-native tool and adding content together with code is not possible. Only UA developers can add content which increases Kyma technical writers' workload.
+- The review process requires using comments in the Help Portal or in a GitHub issue.
+- Publishing documentation is not instant but needs to be triggered manually and requires waiting at least one day for the build.
```

Comment:
we might be able to mitigate that for separate outputs that aren't part of the BTP Core deliverable, but instead maintained and published by us.